src/test/scala/org/broadinstitute/dsde/firecloud/service/UserApiServiceSpec.scala

```diff
@@ -281,69 +232,6 @@ class UserApiServiceSpec extends BaseServiceSpec with RegisterApiService with Us
       }
     }
 
-    "when GET-ting my group membership" - {
```

Comment:
What happened to these tests? Are they unnecessary now or were you planning on moving them somewhere? Maybe a `ManagedGroupServiceSpec`?

Reply:
good catch. i copied almost all of these changes from an old branch of mine here which has the replacement tests i should have carried over: https://github.com/broadinstitute/firecloud-orchestration/compare/mb-sam-groups#diff-20848548f8fd1b7fb280ce41a8c273faR1